packages/chat-client/src/utils/jwtAuth.ts

```diff
@@ -0,0 +1,107 @@
+import { base58btc } from "multiformats/bases/base58";
+import bs58 from "bs58";
+import * as ed25519 from "@noble/ed25519";
+
+interface JwtHeader {
+  typ: string;
+  alg: string;
+}
+
+export const DAY_IN_MS = 86400 * 1000;
+
+export const DID_METHOD_KEY = "key";
+export const DID_DELIMITER = ":";
+export const DID_PREFIX = "did";
+export const DID_METHOD_PKH = "pkh";
+
+export const JWT_DELIMITER = ".";
+
+export const MULTICODEC_ED25519_HEADER = "K36";
+
+export interface InviteKeyClaims {
```

Comment:
Would move this top the top so we have types in a separate section from values/fns